ansible_rulebook/rule_types.py

```diff
@@ -78,6 +78,7 @@ class RuleSet(NamedTuple):
     gather_facts: bool
     uuid: Optional[str] = None
     default_events_ttl: Optional[str] = None
+    match_multiple_rules: Optional[bool] = False
```

Comment:
Minor: type hint is wrong. If the value is not nullable, it should not be optional.
```suggestion
    match_multiple_rules: bool = False
```